src/module.ts

```diff
@@ -2,12 +2,15 @@ import { fileURLToPath } from 'node:url'
 import { defineNuxtModule, addComponentsDir, addPlugin, addServerHandler } from '@nuxt/kit'
 import { join } from 'pathe'
 import defu from 'defu'
+import fs from 'fs'
 
 export interface ModuleOptions {
   /** It is recommended you set the secret key via `runtimeConfig.turnstile.secretKey` or NUXT_TURNSTILE_SECRETKEY */
```

Comment:
Would the recommendation need to be updated? More of a question to Daniel Roe.

Reply:
What do you mean with recommendation as an explanation for me? :)

Reply:
> It is recommended you set the secret key via `runtimeConfig.turnstile.secretKey`

Reply:
Good point. I also think @danielroe should decide which way to pass the secret key should be the recommended one. If passing it directly via the runtimeConfig remains the recommended way then this does not have to be changed. Also the Readme wouldn't have to be updated to explain how the secret key should be passed

Reply:
I think this module should recommend usage without dependencies on external tooling, like Docker, and keep the current recommendation. But the README should still document what options are available to use as it now does in this PR :+1:

Reply:
This thread can be resolved :+1: